src/analyses/goto_check.cpp

```diff
@@ -1813,6 +1831,15 @@ void goto_checkt::goto_check(
         check(i.code.op0());
         // the return value invalidate any assertion
         invalidate(i.code.op0());
+
+        if(has_subexpr(i.code.op0(), [](const exprt &expr) {
+             return expr.id() == ID_r_ok || expr.id() == ID_w_ok;
+           }))
+        {
+          auto rw_ok_cond = rw_ok_check(i.code.op0());
```

Comment:
`to_code_return(code).return_value()`

Reply:
Oops, thank you for keeping me from being lazy.